File: top-100-liked-questions-easy/226. Invert Binary Tree/226.c
struct TreeNode {
    int val;
    struct TreeNode *left;
    struct TreeNode *right;
};

struct TreeNode *invertTree(struct TreeNode *root) {
    if (!root) {
        return root;
    }
    struct TreeNode *tmp = root->left;
    root->left = root->right;
    root->right = tmp;
    if (root->left)
        invertTree(root->left);
    if (root->right)
        invertTree(root->right);
    return root;
}

File: top-100-liked-questions-easy/226. Invert Binary Tree/226.java
public class TreeNode {
    int val;
    TreeNode left;
    TreeNode right;
    TreeNode(int x) { val = x; }
}

class Solution {
    public TreeNode invertTree(TreeNode root) {
        if(root == null){
            return root;
        }
        TreeNode tmp = root.left;
        root.left = root.right;
        root.right = tmp;
        if(root.left != null)
        invertTree(root.left);
        if(root.right!= null)
        invertTree(root.right);
        return root;
    }
}